packages/swing-store/src/snapStore.js

```diff
@@ -119,20 +131,18 @@ export function makeSnapStore(
   }
 
   /**
-   * @param {string} dest basename, relative to root
+   * @param {string} dest absolute path of which root is a prefix
```

Comment:
What's the motivation to switch to absolute paths? I think there is a risk a future change will make the path not prefixed by `root` in which case the rename is not atomic.
Maybe the solution is to extract the `tmpdir` as the dirname of `dest` and ignore `root` ?